protein_ligand_instructions.md

```diff
@@ -0,0 +1,47 @@
+# SAMPL7 Detailed Protein-Ligand (PHIP2) Instructions
+
+Details of the challenge itself, and what information you will submit at each stage, is available in [protein_ligand/README.md](protein_ligand/README.md). This document provides information on how submissions will be handled.
+
+## Due date
+
+Due dates vary by challenge; refer to the main [README.md](README.md) for exact dates.
+
+Your predictions must be uploaded via our [web form](http://sampl-submission.us-west-1.elasticbeanstalk.com/submit) before midnight US Pacific time on the due date. The experimental results will be available as soon as possible after SAMPL closes. Please refer to the below instructions for information on uploading.
+
+You must use the provided templates to upload your predictions. Templates are:
+- [protein_ligand/screening_format.txt](protein_ligand/screening_format.txt) for the virtual screening component
+- (to be updated)
+
+File names must begin with the letters "PHIP2" and the additional details provided in the submission template.
+
+## Multiple submissions
+
+As per our [policy on multiple submissions](https://samplchallenges.github.io/roadmap/submissions/), each participant or organization is allowed only one ranked submission, which must be clearly indicated as such by filling the appropriate field in the submission form. We also accept non-ranked submissions, which we will not formally judge. These allow us to certify that your calculations were done without knowing the answers, but do not receive formal ranking, as discussed at the link above.
+
+If multiple submissions are incorrectly provided as "ranked" by a single participant, we will judge only one of them; likely this will be the first submitted, but it may be a random submission.
+
+## Anonymous participation
+
+Anonymous participation is not allowed.
+
+## Computational methods
+
+You may use any method(s) you like to generate your predictions; e.g. docking, shape-based methods, machine learning models, etc. We only ask that you clearly explain your method in your submission file. Null model submissions are also acceptable.
+
+## Method descriptions
+
+Your method descriptions should give a detailed description of your approach, ideally with enough detail that someone could reproduce the work. These often serve to allow researchers to coordinate on why calculations which seem similar performed quite different in practice, so you should be sure to address how you generated poses, selected protonation states and tautomers if applicable, etc., as well as any method-specific details that, if varied, might result in different performance. Software versions should be provided.
+
```

Comment:
A section must be added here to explain the method categories. Please see my comment in `screening_format.txt`

Reply:
Can you draft?

Reply:
OK. I created a draft. I'll make a commit for it.